src/pow.cpp

```diff
@@ -20,10 +20,25 @@ unsigned int GetNextWorkRequired(const CBlockIndex* pindexLast, const CBlockHead
     {
         if (params.fPowAllowMinDifficultyBlocks)
         {
+            uint8_t target_spacing_multiplier{2};
+
+            // Testnet4
+            if (params.hashGenesisBlock == uint256S("0x000000008d6faa98083fa55742aa82d4ed249bd1bfc3239c706e0a61ef9e3931")) {
+                // Reset limit of Testnet4 is just above difficulty 1M.
+                nProofOfWorkLimit = 0x1b0010c6U;
```

Comment:
What's the reasoning for raising the minimum difficulty to 1 million? This will effectively prevent a developer from being able to move the chain forward without an ASIC.

I'd think that closing the timewarp loophole + the difficulty adjustment loophole should be sufficient to prevent block storms without actually making it more difficult for a dev to mint a low cost block if the chain has stalled.

Reply:
> I'd think that closing the timewarp loophole + the difficulty adjustment loophole should be sufficient to prevent block storms without actually making it more difficult for a dev to mint a low cost block if the chain has stalled.

If that's true then that's fine. It seems multiple people like this ability, see @maflcko's comments above. I'm still a bit confused about the intersection of bugs / features here.

Reply:
It was suggested on the mailing list and it seemed to get some interest, so I added it here. But it has become clear in the discussion here that it seems to be pretty controversial so I will remove it here shortly.

Maybe there is a lower number that is not 1 that most people are happy with like @murchandamus [suggested](https://github.com/bitcoin/bitcoin/pull/29775#issuecomment-2047673316) and I will be happy to add it back in that case but while that is being bikeshedded, I think it's better to remove it.

Reply:
By the way, even if we do implement a higher minimum difficult, imo this should be done in `chainparams.cpp` by setting `consensus.powLimit`. If that makes it harder to grind a genesis block, perhaps that's a good reason not change it.